scripts/performance/benchmark_utils.py

```diff
@@ -0,0 +1,119 @@
+import s3transfer
+import os
+import subprocess
+import uuid
+import shutil
+
+
+TEMP_FILE = 'temp'
+TEMP_KEY = 'temp'
+KB = 1024
+SIZE_SUFFIX = {
+    'kb': 1024,
+    'mb': 1024 ** 2,
+    'gb': 1024 ** 3,
+    'tb': 1024 ** 4,
+    'kib': 1024,
+    'mib': 1024 ** 2,
+    'gib': 1024 ** 3,
+    'tib': 1024 ** 4,
+}
+
+
+def human_readable_to_bytes(value):
+    """Converts a human readable size to bytes.
+    :param value: A string such as "10MB".  If a suffix is not included,
+        then the value is assumed to be an integer representing the size
+        in bytes.
+    :returns: The converted value in bytes as an integer
+    """
+    value = value.lower()
+    if value[-2:] == 'ib':
+        # Assume IEC suffix.
+        suffix = value[-3:].lower()
+    else:
+        suffix = value[-2:].lower()
+    has_size_identifier = (
+        len(value) >= 2 and suffix in SIZE_SUFFIX)
+    if not has_size_identifier:
+        try:
+            return int(value)
+        except ValueError:
+            raise ValueError("Invalid size value: %s" % value)
+    else:
+        multiplier = SIZE_SUFFIX[suffix]
+        return int(value[:-len(suffix)]) * multiplier
+
+
+def summarize(script, directory, filename='summary.txt'):
```

Comment:
I am not sure that I am a fan of adding the ``summarize.txt`` file because if I ever want to run ``summarize`` on that directory again, I would not be able to by doing ``/summarize my-results/*`` assuming my-results is the output directory. I would prefer the directory is reserved to the CSV's.